crates/ethereum-forks/src/forkid.rs

```diff
@@ -297,6 +297,11 @@ impl ForkFilter {
         self.cache.fork_id
     }
 
+    /// Set the current fork id
+    pub fn set_fork_id(&mut self, fork_id: ForkId) {
+        self.cache.fork_id = fork_id;
+    }
+
```

Comment:
I don't think we're using this, right?

setting the forkid on network alone is sufficient, I suppose?

Reply:
I'm using it to set it manually because of the various forks the bsc has, I couldn't make a valid forkhash without